drivers/periph_common/cpuid.c

```diff
@@ -28,15 +28,15 @@
 
 #include "periph/cpuid.h"
 
+typedef struct {
+    uint8_t id[CPUID_LEN];
+} cpu_id_t;
```

Comment:
`cpuid_t`? it's in `cpuid.h`... bikeshedding... I don't care much. ;)

Reply:
ah, but it's not even exported

Reply:
Fixed and squashed